src/index.css

```diff
@@ -282,3 +282,23 @@ code {
 .custom-map-marker-active > .map-marker-pin > .map-marker-text {
   color: #337ab7;
 }
+
+/* Pagination */
```

Comment:
lets try to avoid making custom css classes to do things that can be accomplished with USWDS utility classes or styled components

Reply:
I used the classes that USWDS assigned for this particular component. Do you mean creating inline styling with the utility classes is better?